apisix/admin/init.lua

```diff
@@ -76,7 +76,7 @@ local function check_token(ctx)
 
     local admin
     for i, row in ipairs(local_conf.apisix.admin_key) do
-        if req_token == row.key then
+        if req_token == tostring(row.key) then
```

Comment:
We can wrap the key with `""` to ensure key is a string?
Like `"edd1c9f034335f136f87ad84b625c8f1"`